source/Applications/Advanced/MaskPointCloud/MaskPointCloud.cpp

```diff
@@ -14,17 +14,20 @@ The ZDF file for this sample can be found under the main instructions for Zivid
 
 namespace
 {
-    void visualizePointCloud(const pcl::PointCloud<pcl::PointXYZRGB> &pointCloud)
+    void visualizePointCloud(const boost::shared_ptr<pcl::PointCloud<pcl::PointXYZRGB>> &pointCloud)
     {
-        pcl::PointCloud<pcl::PointXYZRGB>::Ptr cloudPTR(new pcl::PointCloud<pcl::PointXYZRGB>);
-        *cloudPTR = pointCloud;
-        std::cout << "Running PCL visualizer. Block until window closes" << std::endl;
-        pcl::visualization::CloudViewer viewer("Simple Cloud Viewer");
-        viewer.showCloud(cloudPTR);
+        auto viewer = boost::make_shared<pcl::visualization::PCLVisualizer>("Viewer");
```

Comment:
Not sure if we need to replace the `pcl::visualization::CloudViewer` here.
For me it's fine if we show both approaches for visualization in the samples.

Reply:
If you disagree, then we should make sure to update all `CloudViewer` occurences in the repo, for instance https://github.com/zivid/zivid-cpp-samples/blob/5d2d0366740d2c2bb725a6689ebd635251b7c713/source/Applications/Advanced/MultiCamera/StitchByTransformation/StitchByTransformation.cpp#L204.

Reply:
And then we should also make sure to update all includes of the header
`#include <pcl/visualization/cloud_viewer.h>`
to
`#include <pcl/visualization/pcl_visualizer.h>`

Reply:
But as mentioned, I don't mind if we have both ways of visualizing in the repo

Reply:
It actually worked better (view point was better) so I want to keep it here. There are two other samples where we use the other one.

Reply:
I added the correct includes for all PCL samples